DiceKit/FrequencyDistribution.swift

```diff
@@ -190,13 +207,13 @@ extension FrequencyDistribution {
     
     /// This is a special case of `power(x: FrequencyDistribution)`,
     /// for when `x` is `FrequencyDistribution([x: 1])`.
-    public func power(x: Int) -> FrequencyDistribution {
+    public func power(x: Outcome) -> FrequencyDistribution {
         guard x != 0 else { return .multiplicativeIdentity }
         
         // Crappy implementation. Currently O(n). Can be O(log(n)).
         // TODO: Handle negative properly
         var freqDist = self
-        for _ in 1..<abs(x) {
+        for _ in Outcome.multiplicativeIdentity..<abs(x) {
```

Comment:
I've already noticed that this is troublesome. I mean, it works, but it imposes too much on the `Outcome` type (makes it have to be `ForwardIndexType` for example). I'll think on it more.